packages/commonwealth/client/scripts/views/modals/poll_editor_modal.tsx

```diff
@@ -51,13 +49,11 @@ const customDurationOptions = [
 type PollEditorModalProps = {
   onModalClose: () => void;
   thread: Thread;
-  onPollCreate: () => void;
```

Comment:
why did we remove?

Reply:
pre-react legacy, after creating/removing poll, we switched the flag and fetched the polls. Its all gone because of react query

![image](https://github.com/user-attachments/assets/2eadb704-ab05-4462-89ad-a017b52e29db)